values.yaml

```diff
@@ -232,6 +228,13 @@ global:
         memory: "50Mi"
         cpu: "20m"
 
+  # The Docker image for envoy to use as the proxy sidecar when performing
+  # Connect injection or using the mesh/terminating/ingress gateways.
+  #. If using Consul 1.7+, the envoy version must be 1.13+.
```

Comment:
I think we should point to the consul-envoy compatibility [docs](https://www.consul.io/docs/connect/proxies/envoy).